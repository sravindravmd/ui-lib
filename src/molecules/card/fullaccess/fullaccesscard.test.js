import React from 'react'
import { render, cleanup } from '@testing-library/react'
import FullAccessCard from './index'

afterEach(cleanup)

describe('<FullAccessCard/>', () => {
  it('renders FullAccessCard with variant:general', () => {
    const { container } = render(
      <FullAccessCard variant='general' heading='Card' />
    )
    expect(container).toMatchSnapshot()
  })

  it('renders FullAccessCard with variant:withActionItem', () => {
    const { container } = render(
      <FullAccessCard variant='withActionItem' heading='Card' />
    )
    expect(container).toMatchSnapshot()
  })
})
