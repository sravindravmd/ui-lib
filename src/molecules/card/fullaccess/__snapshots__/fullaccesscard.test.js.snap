// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FullAccessCard/> renders FullAccessCard with variant:general 1`] = `
<div>
  <div
    class="sc-bdfBwQ jiStgD"
  >
    <div
      class="sc-gsTCUz bxlNW"
    >
      Card
    </div>
  </div>
</div>
`;

exports[`<FullAccessCard/> renders FullAccessCard with variant:withActionItem 1`] = `
<div>
  <div
    class="sc-bdfBwQ jPLiSs"
  >
    <div
      class="sc-gsTCUz bxlNW"
    >
      Card
    </div>
  </div>
</div>
`;
